README.md

```diff
@@ -26,6 +26,34 @@ There are five primary directories in this repository:
 The book is built with [mdbook](https://github.com/rust-lang/mdBook) and deployed via
 [github pages](https://pages.github.com/).
 
+## Building This Book Locally 
+
+Building the book requires [mdBook], ideally the same version that
+rust-lang/rust uses in [this file][rust-mdbook]. To get it:
+
+[mdBook]: https://github.com/rust-lang-nursery/mdBook
+[rust-mdbook]: https://github.com/rust-lang/rust/blob/master/src/tools/rustbook/Cargo.toml
+
+```bash
+$ cargo install mdbook --vers [version-num]
+```
+
```

Comment:
@NukeManDan thanks for adding the mdbook instruction I think they are good. 

Could you add the `mdbook serve` instruction as well for running the recipe locally.